### PR TITLE
Adding QuickNode to Enterprise Section

### PR DESCRIPTION
**Pull request by Sahilsen**
## Description

Adding QuickNode with a short description in the Enterprise section

## Related Issue

No related issues.

### CODE REVIEW

File: src/content/enterprise/index.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ Some collaborative efforts to make Ethereum enterprise friendly have been made b
 - [Hyperledger Besu](https://www.hyperledger.org/use/besu) _an enterprise focused open-source Ethereum client developed under the Apache 2.0 license and written in Java_
 - [Infura](https://infura.io/) _scalable API access to the Ethereum and IPFS networks_
 - [Provide](https://provide.services/) _infrastructure and APIs for Enterprise Web3 applications_
+- [QuickNode](https://www.quicknode.com/) _provides realiable and fast nodes with high level APIs like NFT API, Token API, etc. To help developers focus on BUIDL_
```
Comment (minimalsm) [inline src/content/enterprise/index.md:66]: Thanks for raising this PR @Sahilsen.

> To help developers focus on BUIDL

I'm not sure the language/tone here makes sense given the target audience of this page. What do you think?

**Review by minimalsm — CHANGES_REQUESTED** [review]

File: src/content/enterprise/index.md (hunk 1 above)
Comment (Sahilsen) [reply src/content/enterprise/index.md:66]: Agreed, I will update this shortly.

thanks for the feedback @minimalsm

**Review by minimalsm — APPROVED** [review]
LGTM :-) thanks for the fixes @Sahilsen

**Comment by Sahilsen** [comment]
> LGTM :-) thanks for the fixes @Sahilsen

Thanks, @minimalsm  just curious that how frequently changes are deployed.

**Pull request merged**